### DISCUSSION TITLE
Strato vServer with Valheim server doesn't show in server list

### DISCUSSION THREAD

**Discussion by Sniphs98** [lloesche/valheim-server-docker#67] [Q&A]
Hey i have a vServer on strato with docker and it runs fine but i dont find the server in the serverlist

Questions
- is this question right hear its more a problem for my salfe and not a discussion or is it good here in case others have the same problem 
- is it enough if I set the port 2456-2458:2456-2458/udp  or do I have to do more to reach it from the outside.
normaly I connect over the domain and the port there like for example with my ts server or minecraft

**Comment by dvvinco** [comment]
Have you tried to add the server as a favorite in the Steam -> Server view? My server didn't show up as well when I used port 2456. Using port 2457 made the server available.

**Reply by Sniphs98** [reply]
the steam server favorite dont works for me but i can try to change the port 
thx for you answer

**Comment by lloesche** [comment]
@Sniphs98 if the UDP ports 2456, 2457 and 2458 are accessible from the Internet then that is fine. Does connecting to the server directly work? The in-game server list has been truncated for a while.

**Reply by Sniphs98** [reply]
i have a ts3 server with docker on the same server i think it is reachable for the internet
